cl/custom_filters/templatetags/extras.py

```diff
@@ -432,3 +432,17 @@ def humanize_number(value):
         formatted = str(num)
 
     return f"{formatted}{abbreviation}"
+
+
+@register.filter
+def has_attr(obj, attr_name):
+    """Return True if obj has attribute attr_name."""
+    return hasattr(obj, attr_name)
+
+
+@register.filter
+def get_attr(obj, attr_name):
+    """Return the value of the attribute attr_name."""
+    if not hasattr(obj, attr_name):
+        return ""
+    return getattr(obj, attr_name)
```

Comment:
The getattr() function accepts an optional default argument. If the specified attribute doesn't exist, this default value is returned. This allows us to refactor the method as follows:

```suggestion
    return getattr(obj, attr_name, "")
```